my-app/backend/src/index.ts

```diff
@@ -0,0 +1,38 @@
+import cors from 'cors'
+import express,{Request,Response} from 'express'
+import * as mongoose from "mongoose";
+import {body, validationResult} from 'express-validator'
+
+
+mongoose
+    .connect('mongodb+srv://admin:admin123@cluster0.ttgwg1i.mongodb.net/?retryWrites=true&w=majority')
+    .then(() => console.log('DB OK'))
+    .catch((err) => console.log('DB ERROR', err));
+
+const app = express()
+const port = 8888
+app.use(cors())
+app.use(express.json())
+
+//Postman Error: read ECONNRESET
+app.post('/register',
+        body('email').isEmail(),
+        body('password').isLength({min:5}),
+        body('fullName').isLength({min:3}),
+        body('avatarUrl').optional().isURL,
+    (req: Request, res: Response) => {
+        const errors = validationResult(req);
+        if (!errors.isEmpty()) {
+            return res.status(400).json({errors: errors.array()});
+        }
+        else return res.json({
```

Comment:
its an one of eslint rule, not a big deal, but you don't need to write "else" here. 

```
if (conditionA) {
   return XYZ
}
return ABC
```

this would work without `else` because you have `return` in your if statement and it is considered to be a cleaner option of writing code. you can read about this rule here https://eslint.org/docs/latest/rules/no-else-return

Reply:
ok, fixed, installed eslint and prettier